src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(action => {
+    switch (action.type) {
+      case ('addProperties'):
```

Comment:
You don't need parentheses in this case, fix all the others too
```suggestion
      case 'addProperties':
```